api/v2.0/swagger.yaml

```diff
@@ -9980,3 +9986,10 @@ definitions:
         items:
           type: string
         description: Links of the vulnerability
+
+  ScanType:
+    type: object
+    properties:
+      scan_type:
+        type: string
+        description: The type of capability, Vulnerability or SBOM.
```

Comment:
we can add an enum for `scan_type`. One example: https://github.com/goharbor/harbor/blob/main/api/v2.0/swagger.yaml#L6769